### PR TITLE
15133 Add Submitting Party Component

### PR DESCRIPTION
**Pull request by ChasBoucher**
*Issue #:* /bcgov/entity#15133

*Description of changes:*

- Add Submitting Party component to Review Screen.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue [hunk 1]
```diff
@@ -139,7 +139,7 @@
               ref="submittingPartyAddress"
               id="submitting-party-address"
               :schema="PartyAddressSchema"
-              :value="submittingParty.address"
+              :value="getMhrRegistrationSubmittingParty.address"
```
Comment (dimak1) [inline ppr-ui/src/components/mhrRegistration/SubmittingParty/MhrSubmittingParty.vue:142]: Why not keep the value as it was `submittingParty`, as it already exists in the `localState` as `getMhrRegistrationSubmittingParty`?

**Review by dimak1 — APPROVED** [review]
Just one comment.

**Comment by dimak1** [comment]
@cameron-eyds I have one comment, which is not critical and can merge.

**Comment by cameron-eyds** [comment]
> @cameron-eyds I have one comment, which is not critical and can merge.

Sounds good sir, lets merge 👍

**Review by cameron-eyds — APPROVED** [review]

**Pull request merged**